diff --git a/node_modules/@expo/vector-icons/build/createIconSet.js b/node_modules/@expo/vector-icons/build/createIconSet.js
index 48982ea..5f75064 100644
--- a/node_modules/@expo/vector-icons/build/createIconSet.js
+++ b/node_modules/@expo/vector-icons/build/createIconSet.js
@@ -53,14 +53,12 @@ export default function (glyphMap, fontName, expoAssetId, fontStyle) {
         _mounted = false;
         _icon;
         state = {
-            fontIsLoaded: Font.isLoaded(fontName),
+            fontIsLoaded: true,
         };
         async componentDidMount() {
             this._mounted = true;
-            if (!this.state.fontIsLoaded) {
-                await Font.loadAsync(font);
-                /* eslint-disable react/no-did-mount-set-state */
-                this._mounted && this.setState({ fontIsLoaded: true });
+            if (!Font.isLoaded(fontName)) {
+                try { await Font.loadAsync(font); } catch (e) { /* font embedded via plugin */ }
             }
         }
         componentWillUnmount() {
